sql/connect/server/pom.xml

```diff
@@ -111,7 +111,6 @@
       <groupId>org.apache.spark</groupId>
       <artifactId>spark-protobuf_${scala.binary.version}</artifactId>
       <version>${project.version}</version>
-      <type>test-jar</type>
```

Comment:
If `<type>test-jar</type>` is used, then during testing it will attempt to fetch `spark-protobuf_2.13-4.0.0-SNAPSHOT-tests.jar`,  but what the tests actually need is `spark-protobuf_2.13-4.0.0-SNAPSHOT.jar`, so this tag should not be added.